docs/src/installation/cross-compile-using-docker.md

```diff
@@ -13,22 +13,26 @@ We can also use `docker` to cross compile on every platform and OS that runs `do
 
     If you are **not** using Docker-Desktop you might have to install [QEMU](https://docs.docker.com/build/building/multi-platform/#install-qemu-manually)
 
-2. Create a docker `compose-file.yml`
+2. Create a docker `docker-compose.yml`
 
     Here we are building a `arm64` binary, so we set `platform: linux/arm64`
 
     ```yaml
     services:
       build-container:
-        image: rust:1.79-bookworm
+        image: rust:1.81-bookworm
```

Comment:
```suggestion
        image: rust:bookworm
```

Maybe we could just use the latest stable release here? Then we don't have to keep that up to date anymore.

Reply:
Then I would use: `rust:1-bookworm` to stick with the major release